Framework/Source/Effects/SkyBox/SkyBox.h

```diff
@@ -54,6 +54,10 @@ namespace Falcor
         */
         static UniquePtr create(Texture::SharedPtr& pTexture, Sampler::SharedPtr pSampler = nullptr, bool renderStereo = false);
         
+        /** create a sky box to be set by the loaded scene
```

Comment:
Oh, this is where it came from. I'm still not sure we can't just use the other `create`